core/src/main/resources/jenkins/security/ApiTokenProperty/resources.css

```diff
@@ -111,6 +111,14 @@
 .token-list .token-list-new-item .new-token-value.visible{
     display: inline-block;
 }
+.token-list .token-list-new-item .copy-new-token{
```

Comment:
The `copy-new-token` does not exist anymore with the use of `copyButton`

Reply:
I missed this part.